tests/sentry/sentry_metrics/querying/data_v2/test_api.py

```diff
@@ -67,4 +74,765 @@ def now(self):
     def ts(self, dt: datetime) -> int:
         return int(dt.timestamp())
 
-    # TODO: add tests once the code is implemented.
+    def mql(self, aggregate: str, metric_mri: str, filters: str = "", group_by: str = "") -> str:
+        query = aggregate + f"({metric_mri})"
+        if filters:
+            query += "{" + filters + "}"
+        if group_by:
+            query += " by" + f"({group_by})"
+
+        return query
+
+    def test_query_with_empty_results(self) -> None:
+        for aggregate, expected_identity in (
```

Comment:
is there a reason for not using `@pytest.mark.parametrize` for this?

Reply:
Parameterize doesn't work on classes from my understanding.